week2/project/src/AppRoutes.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import {
+  BrowserRouter as Router,
+  Route,
+  Routes as RoutesContainer,
+} from "react-router-dom";
+import ProductList from "./components/ProductList/ProductList";
+import ProductDetail from "./components/ProductDetail/ProductDetail";
+import CategoryList from "./components/CategoryList/CategoryList";
+
+const AppRoutes = ({
+  categories,
+  selectedCategory,
+  setSelectedProduct,
+  products,
+  handleCategoryClick,
+}) => (
+  <RoutesContainer>
+    {/* Route for displaying the details of a specific product */}
+    <Route
+      path="/product/:id"
+      element={
+        <ProductDetail
+          products={products}
+          setSelectedProduct={setSelectedProduct}
+        />
+      }
+    />
+
+    {/* Default route for displaying categories and product list */}
+    <Route
+      path="/"
+      element={
+        <React.Fragment>
```

Comment:
You can use the shorthand here, we don't really write `React.Fragment` anymore unless you need to apply styling to it via a class or need to add other attributes (like a `key` when mapping)

```suggestion
        <>
```